src/components/HTMLEngineProvider/HTMLRenderers/EditedRenderer.js

```diff
@@ -17,21 +17,23 @@ function EditedRenderer(props) {
     const defaultRendererProps = _.omit(props, ['TDefaultRenderer', 'style', 'tnode']);
     const isPendingDelete = Boolean(props.tnode.attributes.deleted !== undefined);
     return (
-        <Text
-            // eslint-disable-next-line react/jsx-props-no-spreading
-            {...defaultRendererProps}
-            fontSize={variables.fontSizeSmall}
-            color={themeColors.textSupporting}
-            style={[editedLabelStyles, isPendingDelete && styles.offlineFeedback.deleted]}
-        >
-            {/* Native devices do not support margin between nested text */}
+        <Text>
             <Text
                 selectable={false}
-                style={[styles.w1, styles.userSelectNone]}
+                style={styles.userSelectNone}
             >
                 {' '}
             </Text>
-            {props.translate('reportActionCompose.edited')}
+            <Text
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                {...defaultRendererProps}
+                fontSize={variables.fontSizeSmall}
+                color={themeColors.textSupporting}
+                style={[editedLabelStyles, isPendingDelete && styles.offlineFeedback.deleted]}
+            >
+                {/* Native devices do not support margin between nested text */}
```

Comment:
Do we need this comment here?

Reply:
ok, I see now it was in the wrong place, removing it then